sdk-usage/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+cd ..
+cd sdk
+npx tsc
```

Comment:
If you add the sdk building process here, it would be better to build the sdk entirely using the `build.sh` script, or alternatively, not to build the sdk here at all.

Reply:
probably there should be `npx tsc` somewhere as there is this entry inside `pacakge.json`: `"main": "target/index.js"`